tests/conftest.py

```diff
@@ -11,7 +11,7 @@ def __init__(self, response_dict, status_code, headers=None):
         self.headers = {} if headers is None else headers
 
         if "content-type" not in self.headers:
-            self.headers["content-type"] = "application/json"
+            self.headers["content-type"] = "application/json; charset=utf-8"
```

Comment:
Could we leave this as-is and instead add two tests that set the `Content-Type` to `application/json` and `application/json; charset=utf-8` to ensure that both parse the JSON response as expected?